gwnr/waveform/utils.py

```diff
@@ -259,3 +259,88 @@ def get_isco_frequency(mass1, mass2, spin1z, spin2z):
     f_isco = m_omg_isco / (lal.PI * (mass1 + mass2) * lal.MTSUN_SI)
     print(f"Value of f at ISCO: {f_isco}Hz")
     return f_isco
+
+
+def f_ISCO_spin(mass1, mass2, spin1z, spin2z):
+    """
+    Kerr ISCO frequency fitting formula for aligned-spins binary black holes
+
+    Parameters:
+    ----------
+    mass1, mass2   -- Binary's component masses (in solar masses)
+    spin1z, spin2z   -- z-components of component dimensionless spins (lies in [0,1))
+
+    Returns: Kerr ISCO frequency (in Hz)
+    -------
+    """
+    G = 6.6742 * 1e-11
+    cc = 2.99792458 * 1e8
+    Msun = (G / cc**3) * 1.9891 * 1e30
```

Comment:
Do not define these constants. Use them from `lal`, they are available as `lal.G_SI` and `lal.C_SI` and `lal.MSUN_SI` or `lal.MTSUN_SI` etc.

Reply:
... and `lal` has been imported anyway in this file.